### PR TITLE
new benchmarker/benchmarks; keeps old benchmarks

### PR DESCRIPTION
**Pull request by tabcat**
adds/updates benchmarks and an npm script to use the new benchmarker.
i decided not the do anything with the old benchmark files yet but we can add that as part of the pr if decided.

related: https://github.com/orbitdb/benchmark-runner/pull/32

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "mocha": "^8.4.0",
     "mocha-headless-chrome": "^3.1.0",
     "nyc": "^15.0.1",
-    "orbit-db-benchmark-runner": "^1.0.3",
+    "orbit-db-benchmark-runner": "github:tabcat/benchmark-runner#benchmarker",
```
Comment (aphelionz) [inline package.json:54]: I suppose it's ok to publish a new version of the bechmark runner at this point?

**Review by aphelionz — APPROVED** [review]

**Review by aphelionz — CHANGES_REQUESTED** [review]
Whoops, meant to request changes

File: package.json (hunk 1 above)
Comment (aphelionz) [reply package.json:54]: I published `orbit-db-benchmark-runner@2.0.0` so you can use that here.

**Comment by aphelionz** [comment]
We should probably remove the call to `npm run benchmark` in CI until we have a way to reliably compare performance across builds

**Comment by tabcat** [comment]
> We should probably remove the call to `npm run benchmark` in CI until we have a way to reliably compare performance across builds

I was unaware it was added; where is it added?

**Pull request closed without merging**